src/elements/content-sidebar/versions/VersionsItem.js

```diff
@@ -75,6 +76,13 @@ const VersionsItem = ({
     // Version info helpers
     const versionAction = selectors.getVersionAction(version);
     const versionInteger = versionNumber ? parseInt(versionNumber, 10) : 1;
+    const versionRetention = version.retention;
```

Comment:
Could we destructure `retention` out of `version` with all the other properties above on L73?

Reply:
Maybe we could simplify some of the guard checks by doing something similar to `permissions` and when destructuring `retention` we default it to `{}` and then destructure out `disposition_action, disposition_at, applied_at`